x-pack/test/api_integration/apis/synthetics/services/private_location_test_service.ts

```diff
@@ -71,8 +71,8 @@ export class PrivateLocationTestService {
     }));
 
     await server.savedObjects.create({
-      type: privateLocationsSavedObjectName,
-      id: privateLocationsSavedObjectId,
+      type: legacyPrivateLocationsSavedObjectName,
```

Comment:
We should definitely update these tests to use the new schema since it's directly related to privae locations.